src/teleop/gui/src/components/ArmControls.vue

```diff
@@ -2,11 +2,41 @@
   <div class="wrap">
     <h3>Arm controls</h3>
     <div class="controls">
-      <Checkbox
-        ref="arm-enabled"
-        :name="'Arm Enabled'"
-        @toggle="updateArmEnabled($event)"
-      />
+      <!-- Make opposite option disappear so that we cannot select both -->
+      <!-- Change to radio buttons in the future -->
+      <div>
+        <Checkbox
+          ref="arm-enabled"
+          :name="'Arm Enabled'"
+          @toggle="updateArmEnabled($event)"
+        />
+      </div>
+      <br />
+      <div style="min-width: none">
+        <Checkbox
+          ref="open-loop-enabled"
+          :disabled="servo_enabled"
+          :name="'Open Loop Enabled'"
+          @toggle="open_loop_enabled = $event"
+        />
+      </div>
+      <div>
+        <Checkbox
+          ref="servo-enabled"
+          :disabled="open_loop_enabled"
+          :name="'Servo'"
+          @toggle="servo_enabled = $event"
```

Comment:
Sorry I didn't catch this till now but I'm not sure this is the best interface for these controls. There are three exclusive options, right? "Not enabled", "Openloop", or "servo"? Seems almost like a drop down or probably radio buttons might be better?

Reply:
I could be misunderstanding some part of how this is supposed to work so let me know thoughts

Reply:
Radio buttons probably make more sense now tbh. Also it won't let me respond to your newest comment about the 0 command but I was gonna set up some login jetson/teleop for it so that it gurantees that jetson/teleop won't send anything but zeros. 
> Ok so after thinking through it a bit more, it would probably make more sense to cut the commands off at the jetson_teleop level instead. So we would have the arm enabled checkbox tied to a bool that would then be listened to by jetson_teleop. This way we can access this bool for the purpose of sending/not sending motion planned arm messages as well. I think that this change, however, is beyond the scope of this PR, so I removed the zeroing from the GUI for now and will make the change with the bool in jetson teleop later on.